Makefile

```diff
@@ -103,7 +103,7 @@ vendor-rm-windows:
 .PHONY: vendor-tarball
 vendor-tarball: build vendor
 	VERSION=$(shell bin/netavark --version | cut -f2 -d" ") && \
-	[[ $$VERSION == *-dev ]] && echo "version ends with -dev" && exit 1; \
+	[ $$VERSION = *-dev ] && echo "version ends with -dev" && exit 1; \
```

Comment:
This does not work:
`[ 1.4.1-dev = *-dev ] && echo yes`

Reply:
TIL! That's now fixed by doing a switch statement.